test/utils/test-utils.ts

```diff
@@ -172,10 +172,12 @@ export function getTypeOrmConfig(): TestingConnectionOptions[] {
     try {
 
         try {
-            return require(__dirname + "/../../../../ormconfig.json");
```

Comment:
Can you remove this change - if you want to make it please include it in a separate PR.

Reply:
Hi, thanks for finding time to review the PR.

I was thinking about the same. The problem was that local testing / development for oracledb wasn't friendly and I added a separate instructions how to do it with a couple of cli commands.

I'll create a separate PR for this change with proper docs and remove it from this one.

Reply:
done